### PR TITLE
Ft/codeforafrica contact form

### PR DESCRIPTION
**Pull request by kelvinkipruto**
## Description
This PR implements the contact form block

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots
image
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: apps/codeforafrica/src/lib/data/rest/index.js [hunk 1]
```diff
@@ -37,6 +37,6 @@ export const api = {
 export async function getPageStaticProps(context) {
   const props = await getPageProps(api, context);
   return {
-    props,
+    props: props || {},
```
Comment (koechkevin) [inline apps/codeforafrica/src/lib/data/rest/index.js:40]: Ideally, this should return `404`. No?

**Review by koechkevin — COMMENTED** [review]
@kelvinkipruto

**Comment by kelvinkipruto** [comment]
@koechkevin @kilemensi This is also ready

File: apps/codeforafrica/src/payload/blocks/ContactForm.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const ContactForm = {
+  slug: "contact-form",
+  imageURL: "/images/cms/blocks/contact_form.jpg",
+  imageAltText: "Contact From Code",
```
Comment (kilemensi) [inline apps/codeforafrica/src/payload/blocks/ContactForm.js:4]: What does this even mean?

**Review by kilemensi — APPROVED** [review]

**Pull request merged**